openchemistry/io/orca.py

```diff
@@ -0,0 +1,121 @@
+import json
+import openchemistry as oc
+from avogadro.core import Molecule
+from avogadro.io import FileFormatManager
+from jsonpath_rw import parse
+from .base import BaseReader
+from .constants import EV_TO_J_MOL
+from .utils import _cclib_to_cjson_basis
+
+
+def OrcaReader(BaseReader):
+    """A class to parse orca output files and dump a chemical json file"""
+
+    def read(self, output_file):
+        """Read orca output file"""
+        import openbabel
+        import cclib
+        from pybel import readfile
+
+        molecule = next(readfile('orca', output_file))
+        obconv = openbabel.OBConversion()
+
+        # Generation of SMILES string
```

Comment:
In general we have been saving inchi, inchikey, and smiles on the molecule model (which the calculation refers to). There is no need to add them again in the cjson attached to a calculation.

Reply:
That wasn't clear for me before. I removed those unneeded properties from the cjson object.